src/main/java/se/sundsvall/supportmanagement/service/mapper/MessagingMapper.java

```diff
@@ -60,6 +68,26 @@ public static SmsRequest toSmsRequest(ErrandEntity errandEntity, se.sundsvall.su
 			.sender(smsRequest.getSender());
 	}
 
+	public static List<EmailAttachment> toEmailAttachments(final List<AttachmentEntity> attachments) {
+		return ofNullable(attachments).orElse(emptyList()).stream()
+			.map(MessagingMapper::toEmailAttachment)
+			.toList();
+	}
+
+	static EmailAttachment toEmailAttachment(final AttachmentEntity attachment) {
+		try {
+			byte[] bytes = IOUtils.toByteArray(attachment.getAttachmentData().getFile().getBinaryStream());
+			String encoded = Base64.getEncoder().encodeToString(bytes);

```

Comment:
I think this will be encoded with ISO-8859-1, just a heads up if you maybe want to use UTF-8?